src/PhpBrew/VariantBuilder.php

```diff
@@ -162,7 +162,7 @@ public function __construct()
         $this->variants['mbstring'] = function (ConfigureParameters $params, Build $build, $value) {
             $params = $params->withOption('--enable-mbstring');
 
-            if ($build->compareVersion('5.4') >= 0 && !$build->isDisabledVariant('mbregex')) {
+            if ($build->compareVersion('7.4') >= 0 && !$build->isDisabledVariant('mbregex')) {
```

Comment:
It's not a typo. See the [documentation](https://www.php.net/manual/en/mbstring.installation.php):
> As of PHP 5.4.0, if Oniguruma is already installed on the system, `--with-onig[=DIR]` can be specified to use the installed library.

Reply:
I see, Thanks. I'll close this PR.